web/src/portico/team.ts

```diff
@@ -145,28 +179,37 @@ export default function render_tabs(contributors: Contributor[]): void {
                             github_username: c.github_username,
                             avatar: c.avatar,
                             profile_url: get_profile_url(c, tab_name),
-                            commits: c[repo_name],
+                            commits: repo_list.reduce(
+                                (commits: number, key: RepositoryName) => commits + (c[key] ?? 0),
+                                0,
+                            ),
                         }),
                     )
                     .join("");
 
                 $(`#tab-${CSS.escape(tab_name)} .contributors-grid`).html(html);
-                const contributor_count = filtered_by_repo.length;
-                const hundred_plus_contributor_count = filtered_by_repo.filter((c) => {
-                    const commits = c[repo_name] ?? 0;
+                const contributor_count = filtered_by_tab.length;
+                const hundred_plus_contributor_count = filtered_by_tab.filter((c) => {
+                    const commits = repo_list.reduce(
+                        (commits: number, repo_name: RepositoryName) =>
+                            commits + (c[repo_name] ?? 0),
+                        0,
+                    );
```

Comment:
Similarly here, make a proper object with the data for a tab, and do the reduce once in computing that data, and then features like this can key off of it.